### PR TITLE
Issue #8 - Profile page.

### PR DESCRIPTION
**Pull request by BillSchumacher**
## Description

Added my profile page:

image

## Additional Information

- [x] I have read the [contributing docs](/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING.md) (if this is your first contribution)
- [x] This is not a duplicate of any [existing pull request](https://github.com/scaffold-eth/scaffold-eth-2/pulls)

## Related Issues

Relates to #8 

Your ENS/address:

0x822e7a7209d6C3Da71979AD443D1D3e81E78e2FD

### CODE REVIEW

File: packages/nextjs/app/builders/0x822e7a7209d6C3Da71979AD443D1D3e81E78e2FD/page.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import type { NextPage } from "next";
+import { Address } from "~~/components/scaffold-eth";
+
+const BillPage: NextPage = () => {
+  return (
+    <div className="flex items-center flex-col flex-grow justify-center">
+      <div className="flex gap-10 bg-accent-content/30 rounded-lg">
+        <div className="">
```
Comment (carletex) [inline packages/nextjs/app/builders/0x822e7a7209d6C3Da71979AD443D1D3e81E78e2FD/page.tsx:8]: You can remove the empty className here

File: packages/nextjs/app/builders/0x822e7a7209d6C3Da71979AD443D1D3e81E78e2FD/page.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import type { NextPage } from "next";
+import { Address } from "~~/components/scaffold-eth";
+
+const BillPage: NextPage = () => {
+  return (
+    <div className="flex items-center flex-col flex-grow justify-center">
+      <div className="flex gap-10 bg-accent-content/30 rounded-lg">
```
Comment (carletex) [inline packages/nextjs/app/builders/0x822e7a7209d6C3Da71979AD443D1D3e81E78e2FD/page.tsx:7]: You could add something like: `flex flex-col md:flex-row` to make it look good on mobile too!

**Review by carletex — CHANGES_REQUESTED** [review]
Looking great Bill!!

Just left some tiny review comments

File: packages/nextjs/app/builders/0x822e7a7209d6C3Da71979AD443D1D3e81E78e2FD/page.tsx (hunk 2 above)
Comment (BillSchumacher) [reply packages/nextjs/app/builders/0x822e7a7209d6C3Da71979AD443D1D3e81E78e2FD/page.tsx]: Good call this is much better.

**Review by carletex — APPROVED** [review]
Looking great @BillSchumacher !!

Thanks

**Pull request merged**